plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/BaseMySqlIntegrationSmokeTest.java

```diff
@@ -263,6 +265,72 @@ public void testAggregationPushdown()
         assertThat(query("SELECT approx_set(nationkey) FROM nation")).isNotFullyPushedDown(AggregationNode.class);
     }
 
+    @Test
+    public void testStddevPushdown()
```

Comment:
Stddev -> StddevSamp

Reply:
This tests the whole family of `stddev` functions, I think we can leave this as it is, or split into separate tests.